karma.conf.js

```diff
@@ -1,19 +1,34 @@
 const webpackConfig = require('./config/webpack.karma.config');
 
-module.exports = (config) => {
+module.exports = config => {
```

Comment:
I'm checking in `.vscode` configs like in mongodb-js/vscode and I've updated to `prettier.arrowParens = true` which should match that style. It will all happen automatically on save going forward.